mitosheet/src/jupyter/jupyterUtils.tsx

```diff
@@ -96,4 +96,34 @@ export const getArgs = (analysisToReplayName: string | undefined): Promise<strin
         }
         return resolve([]);
     })
+}
+
+export const writeToNotebookMetadata = (analysisName: string, key: string, value: string): void => {
```

Comment:
I'm not sure based on this function api what I'm supposed to pass to it. Can we:
1. change the name of the function to `writeAnalysisToNotebookMetadata` 
2. update the argument names of `key` and `value` to whatever these things actually are for our use case of writing the analysis to the metadata?

Reply:
A more clear option probably is renaming to `writeMetadataToGeneratedCodeCell`